README.md

```diff
@@ -2,6 +2,9 @@
 
 > This is a database of a vet clinic containing data of animals, their owners and the Clinic's employees.
 
+> Schema Diagram
+![schema Image](./Killy-Davon.png)
```

Comment:
- [ ] According to the project requirements, we are to match our tables to the ERD diagram as seen in the screenshot below
![Screenshot from 2022-10-25 15-45-16](https://user-images.githubusercontent.com/46388113/197820883-bb2c70ea-dc0a-4670-88bf-2052599c6c26.png)

Reply:
@addod19 Thanks for the review. we totally deviated. Thanks for pointing this out to us.